tools/interop-node/feeder/feeder.go

```diff
@@ -114,32 +114,15 @@ func (feeder *Feeder) HandlePrevote(ctx context.Context) error {
 	}
 
 	voteData := types.VoteDataArr{}
-	for _, od := range round.OracleData {
-		switch od.Topic {
-		case oracletypes.OralceTopic_BLOCK:
-			blockDataStr, err := feeder.gatherBlockDataString(od.Sources, uint64(round.Timestamp))
-			if err != nil {
-				feeder.abstainRoundId = round.Id
-				return err
-			}
-
-			voteData = append(voteData, &oracletypes.VoteData{
-				Topic: oracletypes.OralceTopic_BLOCK,
-				Data:  blockDataStr,
-			})
-
-		case oracletypes.OralceTopic_OWNERSHIP:
-			nftDataStr, err := feeder.gatherNftOwnerDataString(od.Sources, uint64(round.Timestamp))
-			if err != nil {
-				feeder.abstainRoundId = round.Id
-				return err
-			}
-			voteData = append(voteData, &oracletypes.VoteData{
-				Topic: oracletypes.OralceTopic_OWNERSHIP,
-				Data:  nftDataStr,
-			})
-		}
+	nftDataStr, err := feeder.gatherNftOwnerDataString(round.Ownerships, uint64(round.Timestamp))
```

Comment:
Please leave `for _, od := range round.OracleData {` loop for future contributor.